packages/app/src/cli/prompts/config.ts

```diff
@@ -25,6 +31,21 @@ export async function selectConfigName(directory: string, defaultName = ''): Pro
   return configName
 }
 
+export async function selectConfigFile(directory: string): Promise<string | undefined> {
+  const files = (await promises.readdir(directory)).filter(
+    (file) => file.startsWith('shopify.app.') && file.endsWith('.toml'),
+  )
```

Comment:
I think we should be able to use `glob` (https://github.com/Shopify/cli/blob/main/packages/cli-kit/src/public/node/fs.ts#L473) here and write something like `await glob(path.join(directory, 'shopify.app*.toml'))`